posts/models.py

```diff
@@ -5,7 +5,9 @@
 
 class Post(models.Model):
 
-    # user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='user')
+    likes = models.ManyToManyField(User, related_name='likes')
```

Comment:
Can you get when a like is created ?

Let's add separate model `Like` and store date created.